src/WebUI/WorkLoads/OtherWorkloadsTable.tsx

```diff
@@ -70,10 +71,10 @@ export class OtherWorkloads extends BaseComponent<IOtherWorkloadsProperties, IOt
         this._store.addListener(WorkloadsEvents.WorkloadPodsFetchedEvent, this._onOrphanPodsFetched);
 
         this._imageDetailsStore.addListener(ImageDetailsEvents.HasImageDetailsEvent, this._setHasImageDetails);
-
         this._actionCreator.getReplicaSets(KubeSummary.getKubeService());
         this._actionCreator.getStatefulSets(KubeSummary.getKubeService());
         this._actionCreator.getDaemonSets(KubeSummary.getKubeService());
+        KubeFactory.telemetryService.scenarioStart(Scenarios.OtherWorkloads);
```

Comment:
same as above for deploymentstables.tsx file.

Reply:
Same as above. Child marks end of scenario started by parent. This helps capturing TTI for entire scenario rather than each component